src/Histogram.ts

```diff
@@ -1,12 +1,8 @@
 import { getColorByChannelIndex } from "./constants/colors.js";
 
-(function() {
-  if (!Math.clamp) {
-    Math.clamp = function(val, cmin, cmax) {
-      return Math.min(Math.max(cmin, val), cmax);
-    };
-  }
-})();
+function clamp(val: number, cmin: number, cmax: number): number {
```

Comment:
Do the Math.clamp -> clamp changes correspond to a js vs. ts difference? (were they needed to pass the linter, or changed for another reason)

Reply:
Yes I think typescript was complaining but also, I have always felt bad about monkeypatching the global Math object.  It's really not great practice.